third-party/cmake/BuildJeMalloc.cmake

```diff
@@ -0,0 +1,18 @@
+ExternalProject_Add(jemalloc
+  PREFIX ${DEPS_BUILD_DIR}
+  URL ${JEMALLOC_URL}
+  DOWNLOAD_DIR ${DEPS_DOWNLOAD_DIR}/jemalloc
+  DOWNLOAD_COMMAND ${CMAKE_COMMAND}
+    -DPREFIX=${DEPS_BUILD_DIR}
+    -DDOWNLOAD_DIR=${DEPS_DOWNLOAD_DIR}/jemalloc
+    -DURL=${JEMALLOC_URL}
+    -DEXPECTED_SHA256=${JEMALLOC_SHA256}
+    -DTARGET=jemalloc
+    -P ${CMAKE_CURRENT_SOURCE_DIR}/cmake/DownloadAndExtractFile.cmake
+  BUILD_IN_SOURCE 1
+  CONFIGURE_COMMAND sh ${DEPS_BUILD_DIR}/src/jemalloc/autogen.sh &&
```

Comment:
This is failing on FreeBSD because the default BUILD_COMMAND invokes make, which is BSD make on FreeBSD.  The install should build everything required, so all you need to do here is add `BUILD_COMMAND ""`.  HTH!